rotkehlchen/chain/ethereum/defi/zerionsdk.py

```diff
@@ -237,7 +240,7 @@ def _query_chain_for_all_balances(self, account: ChecksumEthAddress) -> List:
                     ethereum=self.ethereum,
                     method_name='getBalances',
                     arguments=[account],
-                    call_order=[NodeName.OWN, NodeName.ONEINCH],
```

Comment:
Hmm ... this does not seem right. Don't you see the if check and the comment at the very beginning? There is a reason it's there.

We assume your own node (and from testing we also know 1inch's) do not care about the gas limit or have very high gas limit.

All others do not? I don't really recall, but this is why this is here.

Reply:
Okey where not are of this. By the comment I understood a different thing. Changed it